.github/workflows/integrate.yml

```diff
@@ -36,11 +36,10 @@ jobs:
 
       - name: "Check file permissions"
         run: |
-          test "$(find . -type f -not -path './.git/*' -not -path './tests/integration/bin/*' -executable)" = ""
+          test "$(find . -type f -not -path './.git/*' -executable)" = "./tests/integration/bin/install-wp-tests.sh"
```

Comment:
The whole point of `""` is to list executables.

Reply:
I see now, thanks :)